src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,26 @@
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import { Comments } from '../../types/comments';
+
+type Props = {
+  comment: Comments | null,
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment?.name}</strong>
+
+      {' by '}
+
+      <a
+        className="CommentInfo__email"
+        href={`mailto:${comment?.email}`}
```

Comment:
You should return the message about the mother-in-law there are no comments. Because they are not displayed on the page